### ISSUE TITLE
'ComfyUIManagerLogger' object has no attribute 'isatty'

### ISSUE THREAD

**Issue by koyamakkkkk** [kijai/ComfyUI-DynamiCrafterWrapper#80]
when i try to use dynamiccrafter, this error occured..

Error occurred when executing DownloadAndLoadDynamiCrafterModel:

'ComfyUIManagerLogger' object has no attribute 'isatty'

File "/home/ubuntu/ComfyUI/execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "/home/ubuntu/ComfyUI/execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "/home/ubuntu/ComfyUI/execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "/home/ubuntu/ComfyUI/custom_nodes/ComfyUI-DynamiCrafterWrapper/nodes.py", line 106, in loadmodel
self.model = instantiate_from_config(model_config)
File "/home/ubuntu/ComfyUI/custom_nodes/ComfyUI-DynamiCrafterWrapper/utils/utils.py", line 33, in instantiate_from_config
return get_obj_from_str(config["target"])(**config.get("params", dict()))
File "/home/ubuntu/ComfyUI/custom_nodes/ComfyUI-DynamiCrafterWrapper/utils/utils.py", line 42, in get_obj_from_str
return getattr(importlib.import_module(module, package=package_directory_name), cls)
File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
return _bootstrap._gcd_import(name[level:], package, level)
File "", line 1050, in _gcd_import
File "", line 1027, in _find_and_load
File "", line 1006, in _find_and_load_unlocked
File "", line 688, in _load_unlocked
File "", line 883, in exec_module
File "", line 241, in _call_with_frames_removed
File "/home/ubuntu/ComfyUI/custom_nodes/ComfyUI-DynamiCrafterWrapper/lvdm/models/ddpm3d.py", line 19, in
import pytorch_lightning as pl
File "/opt/tensorflow/lib/python3.10/site-packages/pytorch_lightning/__init__.py", line 27, in
from pytorch_lightning.callbacks import Callback # noqa: E402
File "/opt/tensorflow/lib/python3.10/site-packages/pytorch_lightning/callbacks/__init__.py", line 14, in
from pytorch_lightning.callbacks.batch_size_finder import BatchSizeFinder
File "/opt/tensorflow/lib/python3.10/site-packages/pytorch_lightning/callbacks/batch_size_finder.py", line 26, in
from pytorch_lightning.callbacks.callback import Callback
File "/opt/tensorflow/lib/python3.10/site-packages/pytorch_lightning/callbacks/callback.py", line 22, in
from pytorch_lightning.utilities.types import STEP_OUTPUT
File "/opt/tensorflow/lib/python3.10/site-packages/pytorch_lightning/utilities/types.py", line 41, in
from torchmetrics import Metric
File "/opt/tensorflow/lib/python3.10/site-packages/torchmetrics/__init__.py", line 18, in
import pretty_errors # noqa: F401
File "/opt/tensorflow/lib/python3.10/site-packages/pretty_errors/__init__.py", line 4, in
terminal_is_interactive = sys.stderr.isatty()
<img width="540" alt="スクリーンショット 2024-06-26 2 26 06" src="https://github.com/kijai/ComfyUI-DynamiCrafterWrapper/assets/109265714/f15a119f-ad5a-4770-9363-95b319d9eeb7">
